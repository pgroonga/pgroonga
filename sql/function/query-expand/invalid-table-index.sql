CREATE TABLE synonym_groups (
  synonyms text[]
);

CREATE INDEX synonym_groups_synonyms_index
  ON synonym_groups
  USING pgroonga (synonyms pgroonga_text_array_term_search_ops_v2);

INSERT INTO synonym_groups VALUES (ARRAY['Groonga', 'Senna']);

SELECT pgroonga_query_expand('synonym_groups_synonyms_index',
                             'synonyms',
                             'synonyms',
                             'groonga');

DROP TABLE synonym_groups;
