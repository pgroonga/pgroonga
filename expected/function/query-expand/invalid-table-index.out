CREATE TABLE synonym_groups (
  synonyms text[]
);
CREATE INDEX synonym_groups_synonyms_index
  ON synonym_groups
  USING pgroonga (synonyms pgroonga_text_array_term_search_ops_v2);
INSERT INTO synonym_groups VALUES (ARRAY['Groonga', 'Senna']);
SELECT pgroonga_query_expand('synonym_groups_synonyms_index',
                             'synonyms',
                             'synonyms',
                             'groonga');
ERROR:  pgroonga: [query-expand] the specified table isn't table: <synonym_groups_synonyms_index>: <index>
DROP TABLE synonym_groups;
